features/xslt.yml

```
@@ -0,0 +1,15 @@
+name: XSLT
+description: "Extensible Stylesheet Language Transformations (`XSLT`) is an XML-based language used for the transformation of XML documents into new distinct XML documents. XSLT is often used to convert data between different XML schemas or to convert XML data into web pages or PDF documents."
```

Like JSON and XPath:

```suggestion
description: The `XSLTProcessor` API transforms XML documents into new XML or HTML documents, using XSLT stylesheets. You can use XSLT to convert data between different XML schemas or to convert XML data into web pages or PDF documents. Also known as Extensible Stylesheet Language Transformations.
```